supplier_app/__init__.py

```diff
@@ -3,6 +3,10 @@
 from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
 
+
+CURRENT_STATUS = 1
```

Comment:
We need to clarify for which model we are putting this status. Example: `SUPPLIER_CURRENT_STATUS` or `ENTITY_CURRENT_STATUS`